openfl-tutorials/interactive_api/jax_linear_regression/envoy/envoy_config_2.yaml

```diff
@@ -0,0 +1,13 @@
+params:
```

Comment:
The second envoy confg is not required. You have already mentioned in the README on how to use the config for other envoys